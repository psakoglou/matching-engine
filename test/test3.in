BUY GFD 1000 10 ORDER1
SELL GFD 1001 10 ORDER2
MODIFY ORDER1 BUY 1001 10