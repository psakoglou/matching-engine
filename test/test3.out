TRADE ORDER1 1001 10 ORDER2 1001 10